sample_shavar_list_creation.ini

```diff
@@ -14,6 +14,36 @@ allowlist_url=https://raw.githubusercontent.com/mozilla-services/shavar-list-exc
 # name if s3_upload is enabled.  Leave blank for no local output.
 output=mozpub-track-digest256
 
+# DNT="", all categories except content category
+[tracking-protection-base]
+disconnect_url=https://services.disconnect.me/disconnect-plaintext.json
```

Comment:
The `disconnect_url` for all of these should be the file in our `mozilla-services/shavar-prod-lists` repo just like in https://github.com/mozilla-services/shavar-list-creation-config/blob/04a4dcc47f45be35880cc01f2acc5dafafebabdf/stage.ini#L7

We added an extra step between Disconnect updating their list and our using it to generate new lists for security reasons. In order to attack Firefox users, an attacker has to compromise Disconnect's systems AND the GitHub account of a Mozilla reviewer.